src/openforms/variables/base.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+from openforms.forms.models import FormVariable
+
+
+@dataclass
+class BaseStaticVariable:
+    identifier: str
+
+    def get_initial_value(self, *args, **kwargs):
```

Comment:
you could use `@abc.ABC` and `@abc.abstractmethod` here, and you can add the `# pragma: nocover` comment since measuring coverage for this isn't relevant.